src/utils.ts

```diff
@@ -32,26 +39,60 @@ export const collectCopyTargets = async (
 
       copyTargets.push({
         src: matchedPath,
-        dest: path.join(destDir, rename ?? base)
+        dest: path.join(destDir, rename ?? base),
+        transform
       })
     }
   }
   return copyTargets
 }
 
+async function transformCopy(
+  transform: (content: string, filepath: string) => string,
+  src: string,
+  dest: string
+) {
+  //is directory
+  if (src.indexOf('.') === -1) {
+    const dirs = await fs.readdir(src)
+    dirs.forEach(async fileName => {
+      transformCopy(transform, src + '\\' + fileName, dest + '\\' + fileName)
+    })
+  } else {
+    const s = (await fs.readFile(src)).toString()
+    const content = transform(s, src)
+    await fs.outputFile(dest, content)
+  }
+}
```

Comment:
I do not think this plugin should support `transform` with dir copy.
This feature is not supported with rollup-plugin-copy and it would not be easy to support.
Also your implementation does not support deep directories.
https://github.com/vladshcherbin/rollup-plugin-copy/blob/507bf5e99aa2c6d0d858821e627cb7617a1d9a6d/src/index.js#L28-L30

Reply:
ok,I will pr when i hava time

Reply:
Let me ask you a question, how do you debug in the ts file

Reply:
I usally use `console.log` but you can use debugger as well.

Reply:
Let me ask you a question again , what is this problem?
![image](https://user-images.githubusercontent.com/56909676/164357653-17d52d1f-1098-4690-8c0a-1e405e87767f.png)

Reply:
It seems ok now.